src/Components/Form.js

```diff
@@ -0,0 +1,182 @@
+import axios from 'axios';
+import React,{useState} from 'react';
+import DatePicker from 'react-date-picker'
+import { useNavigate } from 'react-router-dom';
+import { FilePicker } from 'evergreen-ui';
+import "./form.css"
+const Form = () => {
+    const [firstname, setFirstname] = useState("");
```

Comment:
instead of having all these data as an individual variable, we could just use an object and name it "userData" which can have all these details within.

const [firstname, setFirstname] = useState({
    "firstName": "",
    "lastName": "",
    .
    .
    .
});

Reply:
Please follow proper camel casing and pascal casing wherever required.

It should be "firstName" and not "firstname"